src/SpellingBee/StateStorage.py

```diff
@@ -22,16 +23,36 @@ def __Save(dict, fileName):
 def __SearchDict(dict, element):
     dictionaryKeys = dict.keys()
     return element in dictionaryKeys
+
+
+#add comments.!!!!!!!
+def __makeDict(saveStateObj):
+    dict = {'keyLetter': saveStateObj.showKeyLetter(), 'uniqueLetters': saveStateObj.showUniqueLetters(), 
+            'currentScore': saveStateObj.showScore(), 'maxScore' : saveStateObj.showMaxScore(), 
```

Comment:
We'll need to add in the full word list for a puzzle, but we can do that during integration today